src/ui/battle-message-ui-handler.ts

```diff
@@ -227,8 +210,31 @@ export default class BattleMessageUiHandler extends MessageUiHandler {
     });
   }
 
+  topIvs(ivs: integer[], shownIvsCount: integer): Stat[] {
```

Comment:
I'm assuming this is a function to get the top IVs
```suggestion
  getTopIvs(ivs: integer[], shownIvsCount: integer): Stat[] {
```

Reply:
Fixed in next commit